docs/submission/submitting-molecular-data.md

```diff
@@ -60,17 +74,27 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
 
 ### Understanding the Song metadata fields
 
-Song accepts metadata in JSON format (also referred as a `Song payload`), which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
+Song accepts metadata in JSON format (also referred as a `Song payload`), which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
 
-The data fields can be broken down into five main sections: `body`, `experiment`, `samples`, `read groups` and `files`.
+The data fields can be broken down into five main sections: `body`, `experiment`, `samples`, `read groups` and `files`. Each of these sections must be submitted in the payload.
 
-> Note: \*\* denotes a required field. These must be provided as part of the metadata payload or it will immediately fail validation upon submission.
+> Note: \*\* denotes a required field. These **must** be provided as part of the metadata payload or it will immediately fail validation upon submission.
 
 **Body:** The main body of the sequencing payload contains important administrative information.
 
-- `read_group_count`: Number of read groups submitted as part of the raw molecular file.
 - `**studyId`: Corresponds to your ARGO `Program ID`. This is the unique id that is assigned to your program. If you have logged into the ARGO Data Platform, this is the Program Id that you see in the Program Services area. For example, PACA-CA is a Program ID.

```

Comment:
do you want to put this note about where to find Program Id on the above instance of "studyID" instead?

Reply:
you mean in the client configuration?  i dont think so... i think it makes more sense here